src/main/java/duke/io/Storage.java

```diff
@@ -0,0 +1,90 @@
+package duke.io;
+
+import duke.task.*;
```

Comment:
Might want to list out the imports explicitly instead of using wildcard imports since it's not recommended.